src/description/eth_call.md

```diff
@@ -0,0 +1,15 @@
+* **MUST NOT** affect on-chain state 
+
+* **MUST** accept a `from` account that does not exist on-chain and if `from` is not defined it **MUST** be set to `0x0000000000000000000000000000000000000000`
+ 
+* If the `to` is null or not defined on-chain it **MUST** return an empty hex string
+
+* If `max_fee_per_gas` or `max_priority_fee_per_gas` is set the other must be set manually, otherwise they both **MUST** be set to the `gasPrice` or 0 when `gasPrice` is null
+
+* **MUST** accept `gas` that is greater than or equal to the minimum price for the current transaction call and less than `2^64 - 1` or `0xffffffffffffffff` 
+
+* **MUST** consider gas to equal 0 if the `gas` parameter is equal to `null` 
```

Comment:
I don't think we want this.  Instead I think what we want is to specify that either the behavior is undefined, or that the call is executed with XXX gas, or that the call is executed with infinite gas.  I don't know what each client does, but I suspect they all do something different here (some I think even may use block gas limit).

Saying that the call should be executed with 0 gas just means "the call should fail with out of gas immediately" which is unlikely the desired behavior.

Reply:
Yeah, I do not think this is the behaviour. I think the the clients use some specific number for the gas limit in such cases.

Reply:
You are right, I found for geth it sets the gas to 80000000, I do not know for the other clients.